interface/modules/custom_modules/oe-module-prior-authorizations/welcome.php

```diff
@@ -13,7 +13,7 @@
 use OpenEMR\Core\Header;
 
 require_once dirname(__FILE__, 4) . "/globals.php";
-require_once __DIR__ . '/vendor/autoload.php';
+require_once dirname(__FILE__) . '/vendor/autoload.php';
```

Comment:
Is this change intended?

Reply:
I might create a standalone PR to just remove the extra autoload from this module. Small little PRs like that are quick and easy to merge, and that change is worth doing on its own.